packages/auth-helpers/src/lib/craft-auth-sig.ts

```diff
@@ -0,0 +1,50 @@
+import { AuthSig, SignerLike } from '@lit-protocol/types';
+import { ethers } from 'ethers';
+
+/**
+ * Generate an AuthSig object using the signer.
+ *
+ * For more context:
+ * We are only using authSig to generate session sigs. In a newer version, we will stop accepting
+ * authSig all together from the node and will only accept session sigs.
+ *
+ * @param signer the signer must have a "signMessage" method
+ * @param toSign - the message to sign
+ * @param address - (optional) the address of the signer
+ * @returns
+ */
+export const generateAuthSig = async ({
+  signer,
+  toSign,
+  address,
+  algo,
+}: {
+  signer: ethers.Wallet | ethers.Signer | SignerLike;
+  toSign: string;
+  address?: string;
+  algo?: 'LIT_BLS' | 'ed25519';
```

Comment:
I don't think `AuthSig` can be signed with LIT_BLS? Might be worth asking @glitch003 ?

Reply:
@DashKash54 you are correct.  only the network can sign using the Network BLS key in the sign_session_key endpoint.  

but i'm a little confused about this function.  Like this is for creating a local authsig?  and i think for local auth sigs the algo field is empty, right?  we only use it for the session sigs?  so maybe we can remove the algo field?

Reply:
Yea, it's a helper function to craft the shape of an authSig, so we don't have to do `const authSig = {...}` everywhere in the code. The `algo` field is required when using the `sign_session_key` endpoint.